content/platform/overview/registering-an-event-against-a-document-profile-asset/index.md

```diff
@@ -0,0 +1,339 @@
+
+In the event attributes section you should also add the required RKVST event attribute `arc_display_type` together with any other event specific attributes. The Document Profile specifies that `arc_display_type` must be `Publish`.  
+
+```yaml
+---
+steps:
+  - step:
+      action: EVENTS_CREATE
+      description: Record event against My First Document.
+      asset_label: assets/<asset-id>
+    operation: Record
+    behaviour: RecordEvidence
+    asset_attributes: 
+      document_hash_value: 84684c83afd5e9cb3a83439872eae74798979ff5754b15931dbe768092174ec9
+      document_hash_alg: sha256
+      document_version: "2"
+      document_status: Published
+    event_attributes:
+      arc_description: Publish version 2 of My First Document
+      arc_display_type: Publish
+      document_version_authors:
+        - display_name: Dee Author
+          email: dee@writeme.org
+        - display_name: Anne Author
+          email: anne@writeme.org  
+    confirm: true
+```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Add your `event_attributes` and `asset_attributes` as key-value pairs.
+
+Fill out the metadata about your Event; `operation` and `behaviour` detail what class of Event is being performed. By default this should always be `Record` and `RecordEvidence`, respectively.
+
+In the asset attributes section you should include the required RKVST asset attributes as defined by the document profile. The new document version hash value is `document_hash_value`, the hash algorithm is `document_hash_alg` and the `document_status` which must be `Published`.
+
+In the event attributes section you should also add the required RKVST event attribute `arc_display_type` together with any other event specific attributes. The Document Profile specifies that `arc_display_type` must be `Publish`.  
+
+```json
+{
+  "operation": "Record",
+  "behaviour": "RecordEvidence",
+  "asset_attributes": {
+        "document_hash_value":"84684c83afd5e9cb3a83439872eae74798979ff5754b15931dbe768092174ec9",
+        "document_hash_alg":"sha256",
+        "document_version":"2",
+        "document_status":"Published"
+    },
+  "event_attributes": {
+    "arc_description": "Publish version 2 of My First Document",
+    "arc_display_type": "Publish",
+          "document_version_authors": [
+            {
+              "display_name": "Dee Author",
+              "email": "dee@writeme.org"
+            },
+            {
+              "display_name": "Anne Author",
+              "email": "anne@writeme.org"
+            }
+          ]  
+  }
+}
+```
+
+This Event will be POSTed to a specific Asset endpoint when the curl command is run. To do this, you will need to include the correct `assets/<asset-id>` string for the asset in the URL.
+{{< /tab >}}}
+{{< /tabs >}}
+<br>
+
+3. Withdraw Event.
+{{< tabs name="withdraw_event" >}}
+{{{< tab name="UI" >}}
+This option is the final Event in the document lifecycle. When a document is no longer to be used it is withdrawn. 
```

Comment:
Proposed change to this:: _It is possible, during a documents lifespan, that it is no longer required, or should not be used for a number of reasons. In this case, a document can be withdrawn. Withdrawal is an optional, final event in a documents lifecycle._